tools/fetch-team-data.js

```diff
@@ -96,7 +96,7 @@ async function fetchUserProfile(username) {
 			: profile.blog
 				? `https://${profile.blog}`
 				: profile.blog,
-		avatar_url: profile.avatar_url,
+		avatar_url: `${profile.avatar_url.replace("private-", "").split("?")[0]}?v=4`,
```

Comment:
Can we extract this into a shared place (e.g., `tools/utils.js`) as a function so that we don't have to repeat the same logic in multiple files?

Reply:
Done! `tools/utils.js` has been created.